inspector/lib.py

```diff
@@ -52,6 +52,7 @@ class Meta(BaseModel):
 
 class Task(BaseModel):
     vendors_only: set = set()  # run for these vendors only, empty means all
+    servers_only: list[set] = []  # run for these vendor/server pairs only, empty means all
```

Comment:
```suggestion
    servers_only: list[tuple] = []  # run for these vendor/server pairs only, empty means all
```
Down there they're tuples. But it could be `set[tuple]` as well, although it doesn matter, but it's faster to look up.